### ISSUE TITLE
Reading data lists/sets

### ISSUE THREAD

**Issue by olantwin** [dune-lardon/lardon#5]
One can also have a `data-sets` directory that contains lists of validated data files. Typically useful in order to loop over a bunch of files that we will use for a given analysis (like cosmics, calibration, noise, ...). If we would analyse all current TDE files, such a file list could be:
`run00300-00304-test.list`
300_1_cb
300_2_cb
301_1_cb  
301_2_cb
302_1_cb  
302_2_cb
303_1_cb  
303_2_cb
304_1_cb 
304_2_cb
It's easy to remove keys by hand if data quality tells us.

_Originally posted by @YoannKermaidic in https://github.com/dune-lardon/lardon/issues/4#issuecomment-964098617_

**Comment by lzambell** [comment]
I'm not sure I agree with that. The code reconstructs the events wether it's cosmics, calibration, noise. It's at the analysis level, later, to know what you should look at in which file

**Comment by olantwin** [comment]
Good point. At the reconstruction level, we probably just want to process all files for now to debug the technology and identify possible issues.

**Comment by lzambell** [comment]
But we should definitely keep track somewhere of what's going on in the detector once we can see cosmics ! In principle, we should always be able to retrieve the run conditions from the slow control, but a spreadsheet / whatever is also nice to have a quick summary

**Comment by YoannKermaidic** [comment]
Right, perhaps I anticipated a bit the next steps. Although, at some point you need to provide the list of files to process (probably in batch jobs). Having a list of keys can be easier to handle for I/O (like adding extensions at each processing steps) than dealing with explicit file names. Anyhow, certainly not the most urgent topic to discuss =).
